agents/ten_packages/system/ten_ai_base/interface/ten_ai_base/config.py

```diff
@@ -44,6 +46,7 @@ def _init(obj, ten_env: TenEnv):
                         val = ten_env.get_property_float(field.name)
                         setattr(obj, field.name, val)
                     case _:
-                        pass
+                        val = ten_env.get_property_to_json(field.name)
```

Comment:
can we have a more specific type like dict?
better put a comment here.

Reply:
it should support list and dict, but in case the type never match, because the type will be Dict[str, xxxx]

Reply:
it should just pass if type never match.